scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -422,6 +427,23 @@ task ExtractTask {
         --shard-identifier ~{intervals_name} \
         ~{cost_observability_line}
 
+    if [[ -n "~{target_interval_list}" ]]
+    then
+      pre_off_target_vcf="pre_off_target_~{output_file}"
+      mv ~{output_file} ${pre_off_target_vcf}
+      mv ~{output_file}.tbi "${pre_off_target_vcf}.tbi"
+
+      gatk --java-options "-Xmx${memory_mb}m" \
+        IndexFeatureFile \
+          -I ~{target_interval_list}
+
+      gatk --java-options "-Xmx${memory_mb}m" \
+        VariantFiltration \
+          ~{"--filter-not-in-mask --mask-name OUTSIDE_OF_TARGETS --mask-description 'Outside of sequencing target intervals' --mask " + target_interval_list} \
+          -O ~{output_file} \
+          -V ${pre_off_target_vcf}
+    fi
+
```

Comment:
can we not just give an interval list to the GATK extract itself?

Reply:
I don't think this is the same interval list we want to use there, plus the `VariantFiltration` tool writes the specific `OUTSIDE_OF_TARGETS` genotype filter we want with the VCF header and everything.